src/clojure_tictactoe/cli/input/input_getter.clj

```diff
@@ -13,25 +14,53 @@
   (get-user-input)
   nil)
 
-(defn parse-move-input [input]
-  (let [formatted-input (read-string input)]
-    (let [valid-numeral (and
-                          (number? formatted-input)
-                          (< formatted-input 9))]
-      (when valid-numeral
-        formatted-input))))
+(defn format-valid-numeral [number]
+  (- number 1))
 
-(defn get-player-choice []
+(defn format-input [input]
+  (if (or
+        (empty? input)
+        (string/includes? input "#"))
```

Comment:
Can this `or` condition be expressed more explicitly?

Reply:
Can you elaborate what you're looking for?